samples/client/echo_api/java/native/src/test/java/org/openapitools/client/CustomTest.java

```diff
@@ -245,4 +246,25 @@ public void testDefaultValuesSerializationWithIncorrectDefaultValues() throws IO
                     " at [Source: (String)\"{ \"array_string_enum_default\": [\"invalid\"] }\"; line: 1, column: 33] (through reference chain: org.openapitools.client.model.DefaultValue[\"array_string_enum_default\"]->java.util.ArrayList[0])");
         }
     }
+
+
+    /**
+     * Test form parameter(s)
+     *
+     * Test form parameter(s)
+     *
+     * @throws ApiException
+     *          if the Api call fails
+     */
+    @Test
+    public void testFormIntegerBooleanStringTest() throws ApiException {
```

Comment:
Here goes the test, however I don't know how to check response for form data parameters. Can't find anything that I could use in `EchoServerResponseParser`.

Reply:
Right, I think we need to add a field to store the response body (e.g. `body`) and update the following code accordingly:
```
        for (String line : lines) {
            if (firstLine) {
                String[] items = line.split(" ");
                this.method = items[0];
                this.path = items[1];
                this.protocol = items[2];
                firstLine = false;
                continue;
            }

            // store the header key-value pair in headers
            String[] keyValue = line.split(": ");
            if (keyValue.length == 2) { // skip blank line, non key-value pair
                this.headers.put(keyValue[0], keyValue[1]);
            }
            
            // find the blank line and then store the rest into `body`
        }
```

Is that something you can help with ? 🙏

Reply:
Sure, let me try.

Reply:
I finally found (in CI setup) how to run echo server. 
```yaml
        run: |
          git clone https://github.com/wing328/http-echo-server -b openapi-generator-test-server
          (cd http-echo-server && npm install && npm start &)
```
🎉

Reply:
> I finally found (in CI setup) how to run echo server.

We also mention it in https://github.com/OpenAPITools/openapi-generator/wiki/Integration-Tests#echo-server

Reply:
Sorry. I thought I shared the wiki link earlier but I didn't.

Reply:
@wing328 done